src/cuga/backend/cuga_graph/nodes/cuga_flow/process_registry.py

```
@@ -0,0 +1,131 @@
+                continue
+
+            flow_yaml: Optional[Path] = None
+            for yf in config_dir.glob("*.yaml"):
+                try:
+                    with open(yf) as f:
+                        doc = yaml.safe_load(f)
+                    if isinstance(doc, dict) and "flow" in doc:
+                        flow_yaml = yf
+                        break
+                except Exception:
+                    pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)